MegaQuests/src/main/java/com/elmer/megaquests/Quests.java

```diff
@@ -1,79 +1,114 @@
-            ,Material.SKELETON_SKULL, 100, 10,50,  false, true, 0 ),
-    KILLZOMBIE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Zombies"
-            ,Material.ZOMBIE_HEAD, 100, 10, 100, false, true, 0),
-    KILLPLAYER(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Players"
-            ,Material.PLAYER_HEAD, 100, 1, 5,  false, true, 0),
+    MINE_DIAMOND_ORE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Mine Diamonds"
+            , Material.DIAMOND_ORE, 100, 3, 20, false, true, 0),
+    MINE_IRON_ORE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Mine Iron Ore"
+            , Material.IRON_ORE, 100, 10, 150, false, true, 0),
+    MINE_GOLD_ORE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Mine Gold Ore"
+            , Material.GOLD_ORE, 100, 10, 150, false, true, 0),
+    MINE_COAL_ORE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Mine Coal Ore"
+            , Material.COAL_ORE, 100, 10, 150, false, true, 0),
+
     FISHING(ChatColor.GREEN.toString() + ChatColor.BOLD + "Go Fishing"
-            , Material.TROPICAL_FISH, 100,5,20,  false, true, 0),
-    KILLSPIDER(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Spiders"
-            , Material.SPIDER_EYE, 100, 5, 70,  false, true, 0),
-    KILLEVOKER(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Evokers"
-            , Material.TOTEM_OF_UNDYING, 100, 1, 5,  false, true, 0),
-    KILLBLAZE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Blazes"
-            , Material.BLAZE_POWDER, 100, 5, 20,  false, true, 0),
-    MINEIRONORE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Mine Iron Ore"
-            ,Material.IRON_ORE, 100, 10, 150, false, true, 0),
-    MINEGOLDORE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Mine Gold Ore"
-            ,Material.GOLD_ORE, 100, 10, 150, false, true, 0),
-    MINECOALORE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Mine Coal Ore"
-            ,Material.COAL_ORE, 100, 10, 150, false, true, 0);
-
-    private String display;
+            , Material.TROPICAL_FISH, 100, 5, 20, false, true, 0),
+
+    KILL_SKELETON(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Skeletons"
+            , Material.SKELETON_SKULL, 100, 10, 50, false, true, 0),
+    KILL_ZOMBIE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Zombies"
+            , Material.ZOMBIE_HEAD, 100, 10, 100, false, true, 0),
+    KILL_PLAYER(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Players"
+            , Material.PLAYER_HEAD, 100, 1, 5, false, true, 0),
+    KILL_SPIDER(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Spiders"
+            , Material.SPIDER_EYE, 100, 5, 70, false, true, 0),
+    KILL_EVOKER(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Evokers"
+            , Material.TOTEM_OF_UNDYING, 100, 1, 5, false, true, 0),
+    KILL_BLAZE(ChatColor.GREEN.toString() + ChatColor.BOLD + "Kill Blazes"
+            , Material.BLAZE_POWDER, 100, 5, 20, false, true, 0);

```

Comment:
This is generally incorrect programme design in general. 
If you need to extend the functionality, there will be a lot of code duplication in almost all classes. 
I didn't redo this, but I will point out that this approach is not scalable.

Reply:
What other strategy would you suggest? Saving all of the quests in a file or database?